packages/react-server/src/webpack-plugin/test/webpack-plugin.test.ts

```diff
@@ -371,6 +371,18 @@ const universal = {
   resolve: {
     modules: ['node_modules', path.resolve(__dirname, '${basePath}')],
   },
+  module: {
```

Comment:
I had to modify this test webpack config in order to get these tests to pass. Otherwise they would error out and complain about not being able to import from `.mjs` or parse `node` modules. Not sure if this is something we should worry about, maybe someone with more webpack experience can weigh in?

<img width="802" alt="Screen Shot 2020-07-28 at 12 11 56 PM" src="https://user-images.githubusercontent.com/46533681/88691998-8e25c580-d0cb-11ea-8d6f-95566a6073ae.png">

Reply:
I suspect this is because the `package.json`s include `"module": "index.mjs"`. Perhaps webpack takes precedence on the `module` over `main` resolution when it runs builds? 

I'm not an expert on `.mjs` yet, maybe @TheMallen can clarify.

Reply:
I'm thinking we should maybe run a [beta release](https://github.com/Shopify/quilt/blob/master/documentation/guides/release-and-deploy.md#step-1---publish-a-beta-release-for-testing) for these `sewing-kit-next` built packages. This can help verify that

* This doesn't break and dev/prod builds in sewing-kit apps. This seems [supported](https://github.com/Shopify/sewing-kit/blob/master/packages/sewing-kit/src/tools/webpack/config/rules.ts#L426-L430) in sewing-kit already but worth verifying things don't break. 
* Also helps validate tree-shaking benefits of esmodule packages. We can dogfood this in the shrink-ray app and use shrink-ray to verify the bundle reduction from treeshaking